README.md

```diff
@@ -72,6 +66,19 @@ Append `HEAD:master` to the pit push if you are on a branch and want to push tha
 git push git@heroku.com:<app>.git HEAD:master
 ```
 
+Logs database
+-------------
+
+To run the migration commands for the logs database, set `RAILS_ENV=development_logs` as an environment variable.
+
```

Comment:
> And I would recommend running the migrations separately form within the migrations project, as opposed to running them from API. Since that matches what we do in production.

👍 Awesome, I'll look at starting to bundle these in 2.1 and run them from there.